src/codegen/src/main/scala/DocGen.scala

```diff
@@ -38,12 +38,14 @@ object DocGen {
 
   def genRstFiles(): Unit = {
     // Generate a modules.rst file that lists all the .py files to be included in API documentation
-    val moduleString = allFiles(toZipDir, _.getName.endsWith(".py"))
-      .map(f => s"   ${f.getName.dropRight(3)}\n").mkString("")
+    val moduleString = allFiles(toZipDir, _.getName.endsWith(".py")).filterNot(_.getName.startsWith("_"))
+        .filterNot(_.getName.equals("java_params_patch.py"))
```

Comment:
Is this the file with all the class descriptions?  I prefer to have a single place with all of these hard-wired things which will serve as a checklist for things to move into the code (where this filter would get translated into some way to say in the source that no docs should be generated for it).